platform/windows/export/export_plugin.cpp

```diff
@@ -388,17 +388,14 @@ String EditorExportPlatformWindows::get_export_option_warning(const EditorExport
 				if (version_array.size() != 4 || !version_array[0].is_valid_int() ||
 						!version_array[1].is_valid_int() || !version_array[2].is_valid_int() ||
 						!version_array[3].is_valid_int() || file_version.contains_char('-')) {
-					return TTR("Invalid file version.");
+					return TTR("Invalid file version. Proper format is x.x.x.x where x is any int.");
 				}
 			}
 		} else if (p_name == "application/product_version") {
 			String product_version = p_preset->get("application/product_version");
 			if (!product_version.is_empty()) {
-				PackedStringArray version_array = product_version.split(".", false);
-				if (version_array.size() != 4 || !version_array[0].is_valid_int() ||
-						!version_array[1].is_valid_int() || !version_array[2].is_valid_int() ||
-						!version_array[3].is_valid_int() || product_version.contains_char('-')) {
-					return TTR("Invalid product version.");
+				if (product_version[0].is_valid_int()) {
```

Comment:
This doesn't compile:
```
no member named 'is_valid_int' in 'CharProxy<char32_t>'
```

I'm not sure how `CharProxy<char32_t>` works, but `is_digit(production_version[0])` (from `core/string/char_utils.h`) may work.